pkg/resource/resource.go

```diff
@@ -602,6 +602,7 @@ func SetEnvResourceDefaultLabels(env *model.Environment, r *model.Resource) erro
 	case types.EnvironmentDevelopment, types.EnvironmentStaging:
 		r.Labels[types.LabelResourceStoppable] = "true"
 	default:
```

Comment:
i prefer to use `case types.EnvirontmnetProduction` rather than `default`, even if we know the last one is prd at present.